modules/serverless-creating-broker.adoc

```diff
@@ -5,30 +5,12 @@
 [id="serverless-creating-broker_{context}"]
 = Creating a broker manually
 
-To create a broker, you must create a `ServiceAccount` for each namespace and give that `ServiceAccount` the required RBAC permissions.
+The following sections describe how to create a broker.
```

Comment:
I'd say something like "You can manually create a broker for Knative Eventing."

Reply:
@kalexand-rh I have fixed that